common/scripted_triggers/00_lifestyle_triggers.txt

```diff
@@ -82,121 +82,47 @@ can_start_new_lifestyle_tree_trigger = {
 		has_perk = avaricious_perk
 	}
 
-	## Warcraft - Mod triggers
+	## Warcraft
```

Comment:
Why `diplomacy_lifestyle_tree_1_finished_or_not_started_trigger`, etc. aren't added in this file?

Reply:
Done